contracts/modules/SimpleUpgrader.sol

```diff
@@ -53,6 +53,7 @@ contract SimpleUpgrader is BaseModule {
         uint256 i = 0;
         //add new modules
         for (; i < toEnable.length; i++) {
+            require(registry.isRegisteredModule(toEnable[i]), "SU: module is not registered");
```

Comment:
I think we should use the `function isRegisteredModule(address[] calldata _modules) external view returns (bool);` method to check all the modules in one internal call and save some gas.